### DISCUSSION TITLE
Core Metadata V2 API Swagger Documentation

### DISCUSSION THREAD

**Discussion by wangqingpen** [edgexfoundry/community#233] [Q&A]
The documentation in the website cannot be opened, and the Core Metadata V2 API Swagger Documentation can be provided

**Comment by jiekechoo** [comment]
@wangqingpen , it depends on your network.

Try this: 
- download source code from github
- import the file `openapi/v2/core-metadata.yaml` to https://editor.swagger.io/, or use the `mkdocs plugin` swagger-ui-tag to address it

**Reply by wangqingpen** [reply]
I'm sorry all my services are v2 so I can't use v3, I tried both of the ways you said it doesn't work, can you provide documentation in github specific location

**Comment by wangqingpen** [comment]
![image](https://github.com/edgexfoundry/community/assets/72652818/86353f0f-0443-4bd0-bac8-fe1f5c9ad0ad)
It's not a network issue

**Reply by jiekechoo** [reply]
> Try this:
> download source code from github
> import the file openapi/v2/core-metadata.yaml to https://editor.swagger.io/, or use the mkdocs plugin swagger-ui-tag to address it

Try my suggestion. Please...

By the way, you should upgrade EdgeX to v3.

**Comment by cloudxxx8** [comment]
Here is the v2 API docs link
https://docs.edgexfoundry.org/2.3/api/core/Ch-APICoreMetadata/
